src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/afcalc/AlleleFrequencyCalculator.java

```diff
@@ -124,11 +124,14 @@ public AFCalculationResult calculate(final VariantContext vc) {
      * Compute the probability of the alleles segregating given the genotype likelihoods of the samples in vc
      *
      * @param vc the VariantContext holding the alleles and sample information.  The VariantContext
-     *           must have at least 1 alternative allele
+     *           must have at least 1 alternative allele. Hom-ref genotype likelihoods can be approximated
+     *           but
      * @return result (for programming convenience)
      */
     public AFCalculationResult calculate(final VariantContext vc, final int defaultPloidy) {
         Utils.nonNull(vc, "VariantContext cannot be null");
+        Utils.validate(vc.getGenotypes().stream().anyMatch(Genotype::hasLikelihoods),
+                "VariantContext must contain at least one genotype with likelihoods");
```

Comment:
In these error messages, it might be good to include a list of ways that you could end up with genotypes with no likelihoods, to help users debug.